config.default.yaml

```diff
@@ -340,18 +340,38 @@ costs:
 
 
 monte_carlo:
+    # uniform: https://chaospy.readthedocs.io/en/master/api/chaospy.Uniform.html
+    # normal: https://chaospy.readthedocs.io/en/master/api/chaospy.Normal.html
+    # lognormal: https://chaospy.readthedocs.io/en/master/api/chaospy.LogNormal.html
+    # triange: https://chaospy.readthedocs.io/en/master/api/chaospy.Triangle.html
+    # beta: https://chaospy.readthedocs.io/en/master/api/chaospy.Beta.html
+    # gamma: https://chaospy.readthedocs.io/en/master/api/chaospy.Gamma.html
+    # [mean, std] for normal
+    # [shape] for lognormal
+    # [lower_bound, upper_bound] for uniform
+    # [lower_bound, midpoint, upper_bound] for triangle
+    # [alpha, beta] for beta
+    # [shape, scale] for gamma
+    # TODO: Support inputs to simulate outages biggest lines "lines.p_nom_opt.max(): [-3000MW 0MW]""
+    # TODO: Support inputs to simulate outages of biggest power plant "generators.p_nom.max()": [-1000MW 0MW]
+    # ... user can add flexibly more features for the Monte-Carlo sampling
+    # as key: add below the pypsa object for the monte_carlo sampling, "network" is only allowed for filtering!
+    # as value: currently supported format [l_bound, u_bound] or empty [], represent multiplication factors for the object
   options:
     add_to_snakefile: false
```

Comment:
Please add a comment to explicit that when true, monte carlo is enabled

Reply:
Thanks for the observation. I have fixed it in the recent commits.